dockers/docker-fpm-frr/Dockerfile.j2

```diff
@@ -54,4 +55,4 @@ RUN chmod a+x /usr/bin/TSA && \
     chmod a+x /usr/bin/TSB && \
     chmod a+x /usr/bin/TSC
 
-ENTRYPOINT ["/usr/bin/supervisord"]
+ENTRYPOINT /usr/bin/services_start.sh && /usr/bin/supervisord
```

Comment:
Can we run services_start as part of supervisord?

Reply:
There is no such requirement. This most likely would require changes as well as more testing. Supervisord brings no benefit or functionality to FRR.